11/23/2018 01:17:22 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 01:17:23 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 01:17:23 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 01:17:23 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 201 74
11/23/2018 01:17:23 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 01:17:23 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 01:17:23 PM http://127.0.0.1:8000 "POST /api/get-index/ HTTP/1.1" 200 5627
11/23/2018 01:17:23 PM http://127.0.0.1:8000 "PATCH /api/lock_tokens/d72ddfdb35934c108bd059573f5f057a HTTP/1.1" 200 74
11/23/2018 01:17:23 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 01:17:23 PM http://127.0.0.1:8000 "DELETE /api/lock_tokens/d72ddfdb35934c108bd059573f5f057a HTTP/1.1" 204 0
11/23/2018 01:17:32 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 01:17:32 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 01:17:32 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 01:17:33 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 201 74
11/23/2018 01:17:33 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 01:17:33 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 01:17:33 PM http://127.0.0.1:8000 "POST /api/get-index/ HTTP/1.1" 200 5627
11/23/2018 01:17:33 PM http://127.0.0.1:8000 "PATCH /api/lock_tokens/262fa85c019b4fb5a3860d37fab55ab5 HTTP/1.1" 200 74
11/23/2018 01:17:33 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 01:17:33 PM http://127.0.0.1:8000 "DELETE /api/lock_tokens/262fa85c019b4fb5a3860d37fab55ab5 HTTP/1.1" 204 0
11/23/2018 01:18:43 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 01:18:43 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 01:18:43 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 01:18:43 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 201 74
11/23/2018 01:18:43 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 01:18:43 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 01:18:43 PM http://127.0.0.1:8000 "POST /api/get-index/ HTTP/1.1" 200 5627
11/23/2018 01:18:43 PM http://127.0.0.1:8000 "PATCH /api/lock_tokens/04a085c81d174132b98a1a0ed0eb2ddb HTTP/1.1" 200 74
11/23/2018 01:18:44 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 01:18:44 PM http://127.0.0.1:8000 "DELETE /api/lock_tokens/04a085c81d174132b98a1a0ed0eb2ddb HTTP/1.1" 204 0
11/23/2018 01:19:18 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 01:19:18 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 01:19:18 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 01:19:18 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 201 74
11/23/2018 01:19:18 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 01:19:18 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 01:19:18 PM http://127.0.0.1:8000 "POST /api/get-index/ HTTP/1.1" 200 5627
11/23/2018 01:19:18 PM http://127.0.0.1:8000 "PATCH /api/lock_tokens/04f21ee58cc04375b01691a53e46dbdc HTTP/1.1" 200 74
11/23/2018 01:19:19 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 01:19:19 PM http://127.0.0.1:8000 "DELETE /api/lock_tokens/04f21ee58cc04375b01691a53e46dbdc HTTP/1.1" 204 0
11/23/2018 01:19:28 PM Resetting dropped connection: 127.0.0.1
11/23/2018 01:19:28 PM http://127.0.0.1:8000 "PATCH /api/lock_tokens/04f21ee58cc04375b01691a53e46dbdc HTTP/1.1" 404 23
11/23/2018 01:19:28 PM 404 Client Error: Not Found for url: http://127.0.0.1:8000/api/lock_tokens/04f21ee58cc04375b01691a53e46dbdc
11/23/2018 01:21:05 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 01:21:06 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 01:21:06 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 01:21:06 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 201 74
11/23/2018 01:21:06 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 01:21:06 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 01:21:06 PM http://127.0.0.1:8000 "POST /api/get-index/ HTTP/1.1" 200 5627
11/23/2018 01:21:06 PM http://127.0.0.1:8000 "PATCH /api/lock_tokens/68af02348f574431a626d9b330ad948c HTTP/1.1" 200 74
11/23/2018 01:21:06 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 01:21:06 PM http://127.0.0.1:8000 "DELETE /api/lock_tokens/68af02348f574431a626d9b330ad948c HTTP/1.1" 204 0
11/23/2018 01:21:09 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 01:21:09 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 01:21:09 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 01:21:09 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 201 74
11/23/2018 01:21:09 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 01:21:09 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 01:21:09 PM http://127.0.0.1:8000 "POST /api/get-index/ HTTP/1.1" 200 5627
11/23/2018 01:21:09 PM http://127.0.0.1:8000 "PATCH /api/lock_tokens/8fe4da8e39994c4bbef0ac6385e406d3 HTTP/1.1" 200 74
11/23/2018 01:21:09 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 01:21:09 PM http://127.0.0.1:8000 "DELETE /api/lock_tokens/8fe4da8e39994c4bbef0ac6385e406d3 HTTP/1.1" 204 0
11/23/2018 01:28:00 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 01:28:01 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 500 22455
11/23/2018 01:28:31 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 01:28:31 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 500 22455
11/23/2018 02:26:08 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:26:08 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 02:26:08 PM 
Traceback (most recent call last):
  File "startup.py", line 54, in sync
    home_dir = check_home_dir()
  File "startup.py", line 182, in check_home_dir
    raise NoHomeDirException
NoHomeDirException
11/23/2018 02:26:28 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:26:28 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 02:26:41 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:26:41 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 201 74
11/23/2018 02:26:41 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:26:41 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:26:41 PM http://127.0.0.1:8000 "POST /api/get-index/ HTTP/1.1" 200 12
11/23/2018 02:26:41 PM http://127.0.0.1:8000 "PATCH /api/lock_tokens/67b6ba65ede14ed4bf4a37d428b7d4df HTTP/1.1" 200 74
11/23/2018 02:26:45 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:26:45 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 02:26:45 PM Resetting dropped connection: 127.0.0.1
11/23/2018 02:26:45 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 02:26:45 PM Resetting dropped connection: 127.0.0.1
11/23/2018 02:26:45 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 02:26:45 PM Resetting dropped connection: 127.0.0.1
11/23/2018 02:26:45 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 02:26:47 PM Resetting dropped connection: 127.0.0.1
11/23/2018 02:26:50 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:26:50 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 02:26:50 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:26:50 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 02:26:50 PM 
Traceback (most recent call last):
  File "startup.py", line 56, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 98, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 02:26:58 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:26:58 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 02:26:58 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:26:58 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 02:26:58 PM 
Traceback (most recent call last):
  File "startup.py", line 56, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 98, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 02:26:59 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:26:59 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 02:26:59 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:26:59 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 02:26:59 PM 
Traceback (most recent call last):
  File "startup.py", line 56, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 98, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 02:27:00 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:27:00 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 02:27:00 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:27:00 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 02:27:00 PM 
Traceback (most recent call last):
  File "startup.py", line 56, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 98, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 02:27:02 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:27:03 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 02:27:03 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:27:03 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 02:27:03 PM 
Traceback (most recent call last):
  File "startup.py", line 56, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 98, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 02:27:04 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:27:04 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 02:27:04 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:27:04 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 02:27:04 PM 
Traceback (most recent call last):
  File "startup.py", line 56, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 98, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 02:27:05 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:27:05 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 02:27:05 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:27:05 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 02:27:05 PM 
Traceback (most recent call last):
  File "startup.py", line 56, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 98, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 02:27:07 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:27:08 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 02:27:08 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:27:08 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 02:27:08 PM 
Traceback (most recent call last):
  File "startup.py", line 56, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 98, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 02:27:08 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:27:09 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 02:27:09 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:27:09 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 02:27:09 PM 
Traceback (most recent call last):
  File "startup.py", line 56, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 98, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 02:27:10 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:27:10 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 02:27:10 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:27:10 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 02:27:10 PM 
Traceback (most recent call last):
  File "startup.py", line 56, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 98, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 02:27:11 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:27:11 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 02:27:11 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:27:11 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 02:27:11 PM 
Traceback (most recent call last):
  File "startup.py", line 56, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 98, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 02:27:12 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:27:12 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 02:27:12 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:27:12 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 02:27:12 PM 
Traceback (most recent call last):
  File "startup.py", line 56, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 98, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 02:27:14 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:27:14 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 02:27:14 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:27:14 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 02:27:14 PM 
Traceback (most recent call last):
  File "startup.py", line 56, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 98, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 02:27:15 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:27:15 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 02:27:15 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:27:15 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 02:27:15 PM 
Traceback (most recent call last):
  File "startup.py", line 56, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 98, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 02:27:16 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:27:16 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 02:27:16 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:27:16 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 02:27:16 PM 
Traceback (most recent call last):
  File "startup.py", line 56, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 98, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 02:27:17 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:27:17 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 02:27:17 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:27:17 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 02:27:17 PM 
Traceback (most recent call last):
  File "startup.py", line 56, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 98, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 02:27:17 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:27:18 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 02:27:18 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:27:18 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 02:27:18 PM 
Traceback (most recent call last):
  File "startup.py", line 56, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 98, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 02:27:18 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:27:18 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 02:27:18 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:27:18 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 02:27:18 PM 
Traceback (most recent call last):
  File "startup.py", line 56, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 98, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 02:27:19 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:27:19 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 02:27:19 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:27:19 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 02:27:19 PM 
Traceback (most recent call last):
  File "startup.py", line 56, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 98, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 02:27:21 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:27:23 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 02:27:23 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:27:23 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 02:27:23 PM 
Traceback (most recent call last):
  File "startup.py", line 56, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 98, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 02:27:25 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:27:25 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 02:27:25 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:27:25 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 02:27:25 PM 
Traceback (most recent call last):
  File "startup.py", line 56, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 98, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 02:27:26 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:27:26 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 02:27:26 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:27:26 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 02:27:26 PM 
Traceback (most recent call last):
  File "startup.py", line 56, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 98, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 02:27:27 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:27:27 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 02:27:27 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:27:27 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 02:27:27 PM 
Traceback (most recent call last):
  File "startup.py", line 56, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 98, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 02:27:30 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:27:30 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 02:27:30 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:27:30 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 02:27:30 PM 
Traceback (most recent call last):
  File "startup.py", line 56, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 98, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 02:27:31 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:27:31 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 02:27:31 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:27:31 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 02:27:31 PM 
Traceback (most recent call last):
  File "startup.py", line 56, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 98, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 02:27:32 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:27:32 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 02:27:32 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:27:32 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 02:27:32 PM 
Traceback (most recent call last):
  File "startup.py", line 56, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 98, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 02:27:40 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:27:40 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 02:27:40 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:27:40 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 02:27:40 PM 
Traceback (most recent call last):
  File "startup.py", line 56, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 98, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 02:27:40 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:27:41 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 02:27:41 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:27:41 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 02:27:41 PM 
Traceback (most recent call last):
  File "startup.py", line 56, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 98, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 02:27:42 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:27:42 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 02:27:42 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:27:42 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 201 74
11/23/2018 02:27:42 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:27:42 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:27:42 PM http://127.0.0.1:8000 "POST /api/get-index/ HTTP/1.1" 200 428
11/23/2018 02:27:42 PM http://127.0.0.1:8000 "PATCH /api/lock_tokens/1103b0baf95f497db2b6134e24cf206b HTTP/1.1" 200 74
11/23/2018 02:27:52 PM Resetting dropped connection: 127.0.0.1
11/23/2018 02:27:52 PM http://127.0.0.1:8000 "PATCH /api/lock_tokens/1103b0baf95f497db2b6134e24cf206b HTTP/1.1" 200 74
11/23/2018 02:28:02 PM Resetting dropped connection: 127.0.0.1
11/23/2018 02:28:02 PM http://127.0.0.1:8000 "PATCH /api/lock_tokens/1103b0baf95f497db2b6134e24cf206b HTTP/1.1" 200 74
11/23/2018 02:28:04 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:28:04 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 02:28:04 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:28:04 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 02:28:04 PM 
Traceback (most recent call last):
  File "startup.py", line 56, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 98, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 02:28:30 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:28:30 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 02:28:31 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:28:31 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 02:28:31 PM 
Traceback (most recent call last):
  File "startup.py", line 56, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 98, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 02:40:38 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:40:38 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 02:40:38 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:40:38 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 201 74
11/23/2018 02:40:38 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:40:38 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:40:38 PM http://127.0.0.1:8000 "POST /api/get-index/ HTTP/1.1" 200 428
11/23/2018 02:40:38 PM http://127.0.0.1:8000 "PATCH /api/lock_tokens/808427132f04482aa58fcfaf94d44069 HTTP/1.1" 200 74
11/23/2018 02:40:47 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:40:47 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 02:40:47 PM Resetting dropped connection: 127.0.0.1
11/23/2018 02:40:47 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 02:40:47 PM Resetting dropped connection: 127.0.0.1
11/23/2018 02:40:47 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 02:40:47 PM Resetting dropped connection: 127.0.0.1
11/23/2018 02:40:47 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 02:40:47 PM Resetting dropped connection: 127.0.0.1
11/23/2018 02:40:47 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 02:40:47 PM Resetting dropped connection: 127.0.0.1
11/23/2018 02:40:47 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 02:40:47 PM Resetting dropped connection: 127.0.0.1
11/23/2018 02:40:47 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 02:40:47 PM Resetting dropped connection: 127.0.0.1
11/23/2018 02:40:47 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 02:40:47 PM Resetting dropped connection: 127.0.0.1
11/23/2018 02:40:48 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 02:40:48 PM Resetting dropped connection: 127.0.0.1
11/23/2018 02:40:48 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 02:40:48 PM Resetting dropped connection: 127.0.0.1
11/23/2018 02:40:48 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 02:40:48 PM Resetting dropped connection: 127.0.0.1
11/23/2018 02:40:48 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 02:40:48 PM Resetting dropped connection: 127.0.0.1
11/23/2018 02:40:48 PM Resetting dropped connection: 127.0.0.1
11/23/2018 02:40:48 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 02:40:48 PM Resetting dropped connection: 127.0.0.1
11/23/2018 02:40:48 PM http://127.0.0.1:8000 "PATCH /api/lock_tokens/808427132f04482aa58fcfaf94d44069 HTTP/1.1" 200 74
11/23/2018 02:40:48 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 02:40:48 PM Resetting dropped connection: 127.0.0.1
11/23/2018 02:40:48 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 02:40:48 PM Resetting dropped connection: 127.0.0.1
11/23/2018 02:40:48 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 02:40:48 PM Resetting dropped connection: 127.0.0.1
11/23/2018 02:40:48 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 02:40:48 PM Resetting dropped connection: 127.0.0.1
11/23/2018 02:40:48 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 02:40:48 PM Resetting dropped connection: 127.0.0.1
11/23/2018 02:40:48 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 02:40:48 PM Resetting dropped connection: 127.0.0.1
11/23/2018 02:40:48 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 02:40:48 PM Resetting dropped connection: 127.0.0.1
11/23/2018 02:40:48 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 02:40:48 PM Resetting dropped connection: 127.0.0.1
11/23/2018 02:40:48 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 02:40:48 PM Resetting dropped connection: 127.0.0.1
11/23/2018 02:40:49 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 02:40:49 PM Resetting dropped connection: 127.0.0.1
11/23/2018 02:40:49 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 02:40:49 PM Resetting dropped connection: 127.0.0.1
11/23/2018 02:40:49 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 02:40:49 PM Resetting dropped connection: 127.0.0.1
11/23/2018 02:40:49 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 02:40:49 PM Resetting dropped connection: 127.0.0.1
11/23/2018 02:40:49 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 02:40:49 PM Resetting dropped connection: 127.0.0.1
11/23/2018 02:40:49 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 02:40:49 PM Resetting dropped connection: 127.0.0.1
11/23/2018 02:40:49 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 02:40:49 PM Resetting dropped connection: 127.0.0.1
11/23/2018 02:40:49 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 02:40:49 PM Resetting dropped connection: 127.0.0.1
11/23/2018 02:40:49 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 02:40:49 PM Resetting dropped connection: 127.0.0.1
11/23/2018 02:40:49 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 02:40:49 PM Resetting dropped connection: 127.0.0.1
11/23/2018 02:40:49 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 02:40:49 PM Resetting dropped connection: 127.0.0.1
11/23/2018 02:40:49 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 02:40:49 PM Resetting dropped connection: 127.0.0.1
11/23/2018 02:40:49 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 02:40:49 PM Resetting dropped connection: 127.0.0.1
11/23/2018 02:40:49 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 02:40:49 PM Resetting dropped connection: 127.0.0.1
11/23/2018 02:40:50 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 02:40:50 PM Resetting dropped connection: 127.0.0.1
11/23/2018 02:40:50 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 02:40:50 PM Resetting dropped connection: 127.0.0.1
11/23/2018 02:40:50 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 02:40:50 PM Resetting dropped connection: 127.0.0.1
11/23/2018 02:40:50 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 02:40:50 PM Resetting dropped connection: 127.0.0.1
11/23/2018 02:40:50 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 02:40:50 PM Resetting dropped connection: 127.0.0.1
11/23/2018 02:40:50 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 02:40:50 PM Resetting dropped connection: 127.0.0.1
11/23/2018 02:40:50 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 02:40:50 PM Resetting dropped connection: 127.0.0.1
11/23/2018 02:40:50 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 02:40:50 PM Resetting dropped connection: 127.0.0.1
11/23/2018 02:40:50 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 02:40:50 PM Resetting dropped connection: 127.0.0.1
11/23/2018 02:40:50 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 02:40:50 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:40:50 PM http://127.0.0.1:8000 "DELETE /api/lock_tokens/808427132f04482aa58fcfaf94d44069 HTTP/1.1" 204 0
11/23/2018 02:41:06 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:41:06 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 02:41:06 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:41:06 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 201 74
11/23/2018 02:41:06 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:41:06 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:41:06 PM http://127.0.0.1:8000 "POST /api/get-index/ HTTP/1.1" 200 5627
11/23/2018 02:41:06 PM http://127.0.0.1:8000 "PATCH /api/lock_tokens/4fd5f1bd37e248fe99f402ab789131e5 HTTP/1.1" 200 74
11/23/2018 02:41:08 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:41:08 PM http://127.0.0.1:8000 "POST /api/delete/ HTTP/1.1" 200 24
11/23/2018 02:41:08 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 02:41:08 PM http://127.0.0.1:8000 "DELETE /api/lock_tokens/4fd5f1bd37e248fe99f402ab789131e5 HTTP/1.1" 204 0
11/23/2018 03:51:55 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 03:51:55 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 03:51:55 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 03:51:55 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 201 74
11/23/2018 03:51:55 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 03:51:55 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 03:51:55 PM http://127.0.0.1:8000 "POST /api/get-index/ HTTP/1.1" 200 12
11/23/2018 03:51:55 PM http://127.0.0.1:8000 "PATCH /api/lock_tokens/83854a4fb0b645259eea86726500467c HTTP/1.1" 200 74
11/23/2018 03:51:58 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 03:51:58 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 03:51:58 PM Resetting dropped connection: 127.0.0.1
11/23/2018 03:51:58 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 03:51:58 PM Resetting dropped connection: 127.0.0.1
11/23/2018 03:51:58 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 03:51:58 PM Resetting dropped connection: 127.0.0.1
11/23/2018 03:51:58 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 03:52:00 PM Resetting dropped connection: 127.0.0.1
11/23/2018 03:52:03 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 03:52:03 PM Resetting dropped connection: 127.0.0.1
11/23/2018 03:52:03 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 03:52:03 PM Resetting dropped connection: 127.0.0.1
11/23/2018 03:52:03 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 03:52:03 PM Resetting dropped connection: 127.0.0.1
11/23/2018 03:52:03 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 03:52:03 PM Resetting dropped connection: 127.0.0.1
11/23/2018 03:52:03 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 03:52:03 PM Resetting dropped connection: 127.0.0.1
11/23/2018 03:52:03 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 03:52:03 PM Resetting dropped connection: 127.0.0.1
11/23/2018 03:52:03 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 03:52:03 PM Resetting dropped connection: 127.0.0.1
11/23/2018 03:52:03 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 03:52:03 PM Resetting dropped connection: 127.0.0.1
11/23/2018 03:52:03 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 03:52:03 PM Resetting dropped connection: 127.0.0.1
11/23/2018 03:52:03 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 03:52:03 PM Resetting dropped connection: 127.0.0.1
11/23/2018 03:52:04 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 03:52:04 PM Resetting dropped connection: 127.0.0.1
11/23/2018 03:52:04 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 03:52:04 PM Resetting dropped connection: 127.0.0.1
11/23/2018 03:52:04 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 03:52:04 PM Resetting dropped connection: 127.0.0.1
11/23/2018 03:52:04 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 03:52:04 PM Resetting dropped connection: 127.0.0.1
11/23/2018 03:52:04 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 03:52:04 PM Resetting dropped connection: 127.0.0.1
11/23/2018 03:52:04 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 03:52:04 PM Resetting dropped connection: 127.0.0.1
11/23/2018 03:52:04 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 03:52:04 PM Resetting dropped connection: 127.0.0.1
11/23/2018 03:52:04 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 03:52:04 PM Resetting dropped connection: 127.0.0.1
11/23/2018 03:52:04 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 03:52:04 PM Resetting dropped connection: 127.0.0.1
11/23/2018 03:52:04 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 03:52:04 PM Resetting dropped connection: 127.0.0.1
11/23/2018 03:52:04 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 03:52:04 PM Resetting dropped connection: 127.0.0.1
11/23/2018 03:52:04 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 03:52:04 PM Resetting dropped connection: 127.0.0.1
11/23/2018 03:52:04 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 03:52:04 PM Resetting dropped connection: 127.0.0.1
11/23/2018 03:52:04 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 03:52:04 PM Resetting dropped connection: 127.0.0.1
11/23/2018 03:52:04 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 03:52:04 PM Resetting dropped connection: 127.0.0.1
11/23/2018 03:52:05 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 03:52:05 PM Resetting dropped connection: 127.0.0.1
11/23/2018 03:52:05 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 03:52:05 PM Resetting dropped connection: 127.0.0.1
11/23/2018 03:52:05 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 03:52:05 PM Resetting dropped connection: 127.0.0.1
11/23/2018 03:52:05 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 03:52:05 PM Resetting dropped connection: 127.0.0.1
11/23/2018 03:52:05 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 03:52:05 PM Resetting dropped connection: 127.0.0.1
11/23/2018 03:52:05 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 03:52:05 PM Resetting dropped connection: 127.0.0.1
11/23/2018 03:52:05 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 03:52:05 PM Resetting dropped connection: 127.0.0.1
11/23/2018 03:52:05 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 03:52:05 PM Resetting dropped connection: 127.0.0.1
11/23/2018 03:52:05 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 03:52:05 PM Resetting dropped connection: 127.0.0.1
11/23/2018 03:52:05 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 03:52:05 PM Resetting dropped connection: 127.0.0.1
11/23/2018 03:52:05 PM Resetting dropped connection: 127.0.0.1
11/23/2018 03:52:05 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 03:52:05 PM Resetting dropped connection: 127.0.0.1
11/23/2018 03:52:05 PM http://127.0.0.1:8000 "PATCH /api/lock_tokens/83854a4fb0b645259eea86726500467c HTTP/1.1" 200 74
11/23/2018 03:52:05 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 03:52:05 PM Resetting dropped connection: 127.0.0.1
11/23/2018 03:52:05 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 03:52:05 PM Resetting dropped connection: 127.0.0.1
11/23/2018 03:52:06 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 03:52:06 PM Resetting dropped connection: 127.0.0.1
11/23/2018 03:52:06 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 03:52:06 PM Resetting dropped connection: 127.0.0.1
11/23/2018 03:52:06 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 03:52:06 PM Resetting dropped connection: 127.0.0.1
11/23/2018 03:52:06 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 03:52:06 PM Resetting dropped connection: 127.0.0.1
11/23/2018 03:52:06 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 03:52:06 PM Resetting dropped connection: 127.0.0.1
11/23/2018 03:52:06 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 03:52:06 PM Resetting dropped connection: 127.0.0.1
11/23/2018 03:52:06 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 03:52:06 PM Resetting dropped connection: 127.0.0.1
11/23/2018 03:52:06 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 03:52:06 PM Resetting dropped connection: 127.0.0.1
11/23/2018 03:52:06 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 03:52:06 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 03:52:06 PM http://127.0.0.1:8000 "DELETE /api/lock_tokens/83854a4fb0b645259eea86726500467c HTTP/1.1" 204 0
11/23/2018 04:29:36 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 04:29:36 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 04:29:37 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 04:29:37 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 201 74
11/23/2018 04:29:37 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 04:29:37 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 04:29:37 PM http://127.0.0.1:8000 "POST /api/get-index/ HTTP/1.1" 200 5627
11/23/2018 04:29:37 PM http://127.0.0.1:8000 "PATCH /api/lock_tokens/2c2b1e30fa394890be42c551e4f91a9d HTTP/1.1" 200 74
11/23/2018 04:29:42 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 04:29:42 PM http://127.0.0.1:8000 "POST /api/delete/ HTTP/1.1" 200 24
11/23/2018 04:29:42 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 04:29:42 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 04:29:42 PM Resetting dropped connection: 127.0.0.1
11/23/2018 04:29:42 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 04:29:42 PM Resetting dropped connection: 127.0.0.1
11/23/2018 04:29:42 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 04:29:42 PM Resetting dropped connection: 127.0.0.1
11/23/2018 04:29:42 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 04:29:42 PM Resetting dropped connection: 127.0.0.1
11/23/2018 04:29:42 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 04:29:42 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 04:29:42 PM http://127.0.0.1:8000 "DELETE /api/lock_tokens/2c2b1e30fa394890be42c551e4f91a9d HTTP/1.1" 204 0
11/23/2018 04:42:58 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 04:42:58 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 04:42:58 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 04:42:58 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 201 74
11/23/2018 04:42:58 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 04:42:58 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 04:42:58 PM http://127.0.0.1:8000 "POST /api/get-index/ HTTP/1.1" 200 415
11/23/2018 04:42:58 PM http://127.0.0.1:8000 "PATCH /api/lock_tokens/f1651100d937410f9f9d6890c0e16bf8 HTTP/1.1" 200 74
11/23/2018 04:43:03 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 04:43:03 PM http://127.0.0.1:8000 "POST /api/delete/ HTTP/1.1" 200 24
11/23/2018 04:43:03 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 04:43:03 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 04:43:03 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 04:43:03 PM http://127.0.0.1:8000 "DELETE /api/lock_tokens/f1651100d937410f9f9d6890c0e16bf8 HTTP/1.1" 204 0
11/23/2018 04:43:06 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 04:43:06 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 04:43:06 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 04:43:06 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 201 74
11/23/2018 04:43:06 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 04:43:06 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 04:43:06 PM http://127.0.0.1:8000 "POST /api/get-index/ HTTP/1.1" 200 84
11/23/2018 04:43:06 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 04:43:07 PM http://127.0.0.1:8000 "PATCH /api/lock_tokens/42cf50d4e875458fb9e2f49b10f1f8ad HTTP/1.1" 200 74
11/23/2018 04:43:07 PM http://127.0.0.1:8000 "DELETE /api/lock_tokens/42cf50d4e875458fb9e2f49b10f1f8ad HTTP/1.1" 204 0
11/23/2018 05:10:07 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:10:07 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:10:07 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:10:07 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 201 74
11/23/2018 05:10:07 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:10:07 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:10:08 PM http://127.0.0.1:8000 "POST /api/get-index/ HTTP/1.1" 500 19498
11/23/2018 05:15:16 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:15:16 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:15:16 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:15:16 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 201 74
11/23/2018 05:15:16 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:15:16 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:15:16 PM http://127.0.0.1:8000 "POST /api/get-index/ HTTP/1.1" 200 12
11/23/2018 05:15:16 PM http://127.0.0.1:8000 "PATCH /api/lock_tokens/bd05d2523f3c48f5af2c724b64af6e4b HTTP/1.1" 200 74
11/23/2018 05:15:19 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:15:19 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 400 31
11/23/2018 05:15:19 PM {"Error":"File already exists"}
11/23/2018 05:15:19 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:15:19 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 400 31
11/23/2018 05:15:19 PM {"Error":"File already exists"}
11/23/2018 05:15:19 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:15:19 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 400 31
11/23/2018 05:15:19 PM {"Error":"File already exists"}
11/23/2018 05:15:19 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:15:19 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 400 31
11/23/2018 05:15:19 PM {"Error":"File already exists"}
11/23/2018 05:15:21 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:15:23 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 400 31
11/23/2018 05:15:23 PM {"Error":"File already exists"}
11/23/2018 05:15:53 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:15:53 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:15:53 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:15:53 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:15:53 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:15:55 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:15:55 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:15:55 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:15:55 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:15:55 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:16:40 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:16:40 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:16:40 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:16:40 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 201 74
11/23/2018 05:16:40 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:16:40 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:16:40 PM http://127.0.0.1:8000 "POST /api/get-index/ HTTP/1.1" 200 12
11/23/2018 05:16:40 PM http://127.0.0.1:8000 "PATCH /api/lock_tokens/be879d724e78408e87cae4ec2670c7cf HTTP/1.1" 200 74
11/23/2018 05:16:43 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:16:43 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 400 31
11/23/2018 05:16:43 PM {"Error":"File already exists"}
11/23/2018 05:16:43 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:16:43 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 400 31
11/23/2018 05:16:43 PM {"Error":"File already exists"}
11/23/2018 05:16:43 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:16:43 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 400 31
11/23/2018 05:16:43 PM {"Error":"File already exists"}
11/23/2018 05:16:43 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:16:43 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 400 31
11/23/2018 05:16:43 PM {"Error":"File already exists"}
11/23/2018 05:16:45 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:16:47 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 400 31
11/23/2018 05:16:47 PM {"Error":"File already exists"}
11/23/2018 05:17:29 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:17:29 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:17:29 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:17:29 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:17:29 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:17:31 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:17:31 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:17:31 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:17:31 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:17:31 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:17:32 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:17:32 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:17:32 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:17:32 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:17:32 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:17:33 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:17:33 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:17:33 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:17:33 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:17:33 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:17:34 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:17:34 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:17:34 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:17:34 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:17:34 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:17:35 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:17:35 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:17:35 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:17:35 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:17:35 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:17:37 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:17:37 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:17:37 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:17:37 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:17:37 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:17:42 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:17:42 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:17:42 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:17:42 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 201 74
11/23/2018 05:17:42 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:17:42 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:17:42 PM http://127.0.0.1:8000 "POST /api/get-index/ HTTP/1.1" 200 12
11/23/2018 05:17:42 PM http://127.0.0.1:8000 "PATCH /api/lock_tokens/57a9213cc68747d0b84d1d5f3cf8e0e4 HTTP/1.1" 200 74
11/23/2018 05:17:44 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:17:44 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 400 31
11/23/2018 05:17:44 PM {"Error":"File already exists"}
11/23/2018 05:17:44 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:17:44 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 400 31
11/23/2018 05:17:44 PM {"Error":"File already exists"}
11/23/2018 05:17:44 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:17:44 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 400 31
11/23/2018 05:17:44 PM {"Error":"File already exists"}
11/23/2018 05:17:44 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:17:44 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 400 31
11/23/2018 05:17:44 PM {"Error":"File already exists"}
11/23/2018 05:17:46 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:17:48 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 400 31
11/23/2018 05:17:48 PM {"Error":"File already exists"}
11/23/2018 05:21:56 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:21:56 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:21:56 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:21:56 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 201 74
11/23/2018 05:21:56 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:21:56 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:21:56 PM http://127.0.0.1:8000 "POST /api/get-index/ HTTP/1.1" 200 12
11/23/2018 05:21:56 PM http://127.0.0.1:8000 "PATCH /api/lock_tokens/58b1b7b9bd4b490ab48a64a129db6f6e HTTP/1.1" 200 74
11/23/2018 05:21:59 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:21:59 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 400 31
11/23/2018 05:21:59 PM {"Error":"File already exists"}
11/23/2018 05:21:59 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:21:59 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 400 31
11/23/2018 05:21:59 PM {"Error":"File already exists"}
11/23/2018 05:21:59 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:21:59 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 400 31
11/23/2018 05:21:59 PM {"Error":"File already exists"}
11/23/2018 05:21:59 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:21:59 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 400 31
11/23/2018 05:21:59 PM {"Error":"File already exists"}
11/23/2018 05:22:01 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:22:02 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 400 31
11/23/2018 05:22:02 PM {"Error":"File already exists"}
11/23/2018 05:26:16 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:26:17 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:26:17 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:26:17 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 201 74
11/23/2018 05:26:17 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:26:17 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:26:17 PM http://127.0.0.1:8000 "POST /api/get-index/ HTTP/1.1" 200 12
11/23/2018 05:26:17 PM http://127.0.0.1:8000 "PATCH /api/lock_tokens/342177c54eed4f7c911bc3e5c9b89eee HTTP/1.1" 200 74
11/23/2018 05:26:19 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:26:19 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 400 31
11/23/2018 05:26:19 PM {"Error":"File already exists"}
11/23/2018 05:26:19 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:26:19 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 400 31
11/23/2018 05:26:19 PM {"Error":"File already exists"}
11/23/2018 05:26:19 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:26:19 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 400 31
11/23/2018 05:26:19 PM {"Error":"File already exists"}
11/23/2018 05:26:19 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:26:19 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 400 31
11/23/2018 05:26:19 PM {"Error":"File already exists"}
11/23/2018 05:26:21 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:26:23 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 400 31
11/23/2018 05:26:23 PM {"Error":"File already exists"}
11/23/2018 05:27:44 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:27:45 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:27:45 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:27:45 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 201 74
11/23/2018 05:27:45 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:27:45 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:27:45 PM http://127.0.0.1:8000 "POST /api/get-index/ HTTP/1.1" 200 12
11/23/2018 05:27:45 PM http://127.0.0.1:8000 "PATCH /api/lock_tokens/afcdb4ddd55b4b04b4efd6132b4321a4 HTTP/1.1" 200 74
11/23/2018 05:27:48 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:27:48 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 200 47
11/23/2018 05:27:48 PM {"error":"unexpected error, please try again."}
11/23/2018 05:27:48 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:27:48 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 200 47
11/23/2018 05:27:48 PM {"error":"unexpected error, please try again."}
11/23/2018 05:27:48 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:27:48 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 200 47
11/23/2018 05:27:48 PM {"error":"unexpected error, please try again."}
11/23/2018 05:27:48 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:27:49 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 200 47
11/23/2018 05:27:49 PM {"error":"unexpected error, please try again."}
11/23/2018 05:27:50 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:27:52 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 200 47
11/23/2018 05:27:52 PM {"error":"unexpected error, please try again."}
11/23/2018 05:28:25 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:25 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:28:25 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:25 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:28:25 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:28:27 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:27 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:28:27 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:27 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:28:27 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:28:27 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:28 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:28:28 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:28 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:28:28 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:28:28 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:28 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:28:28 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:28 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:28:28 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:28:29 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:29 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:28:29 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:29 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:28:29 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:28:30 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:30 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:28:30 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:30 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:28:30 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:28:30 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:30 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:28:30 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:30 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:28:30 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:28:31 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:31 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:28:31 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:31 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:28:31 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:28:34 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:34 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:28:34 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:34 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:28:34 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:28:38 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:38 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:28:38 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:38 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:28:38 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:28:39 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:39 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:28:39 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:39 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:28:39 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:28:39 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:39 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:28:40 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:40 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:28:40 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:28:40 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:40 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:28:40 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:40 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:28:40 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:28:41 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:41 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:28:41 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:41 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:28:41 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:28:41 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:42 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:28:42 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:42 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:28:42 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:28:42 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:42 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:28:42 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:42 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:28:42 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:28:43 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:43 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:28:43 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:43 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:28:43 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:28:44 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:44 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:28:44 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:44 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:28:44 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:28:46 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:46 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:28:46 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:46 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 201 74
11/23/2018 05:28:46 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:46 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:46 PM http://127.0.0.1:8000 "POST /api/get-index/ HTTP/1.1" 200 12
11/23/2018 05:28:46 PM http://127.0.0.1:8000 "PATCH /api/lock_tokens/76b6d4ab50aa44ebbaefc762eebf1c0b HTTP/1.1" 200 74
11/23/2018 05:28:52 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:52 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:28:52 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:52 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:28:52 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:28:53 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:53 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:28:53 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:53 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:28:53 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:28:54 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:55 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:28:55 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:55 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:28:55 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:28:56 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:56 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:28:56 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:56 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:28:56 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:28:56 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:57 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:28:57 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:57 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:28:57 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:28:57 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:57 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:28:57 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:57 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:28:57 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:28:58 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:58 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:28:58 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:58 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:28:58 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:28:59 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:59 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:28:59 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:28:59 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:28:59 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:29:01 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:01 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:29:01 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:01 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:29:01 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:29:02 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:02 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:29:02 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:02 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:29:02 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:29:03 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:03 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:29:03 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:03 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:29:03 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:29:04 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:04 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:29:04 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:04 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:29:04 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:29:06 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:06 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:29:06 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:06 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:29:06 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:29:07 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:07 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:29:07 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:07 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:29:07 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:29:08 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:08 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:29:08 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:09 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:29:09 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:29:10 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:10 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:29:10 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:10 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:29:10 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:29:11 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:11 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:29:11 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:11 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:29:11 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:29:13 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:13 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:29:13 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:13 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:29:13 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:29:15 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:15 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:29:15 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:15 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:29:15 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:29:16 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:16 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:29:16 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:16 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:29:16 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:29:17 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:18 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:29:18 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:18 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:29:18 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:29:19 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:19 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:29:19 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:19 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:29:19 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:29:20 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:20 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:29:20 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:20 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:29:20 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:29:21 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:21 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:29:21 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:21 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:29:21 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:29:22 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:22 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:29:22 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:22 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:29:22 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:29:24 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:24 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:29:24 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:24 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:29:24 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:29:25 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:25 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:29:25 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:25 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:29:25 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:29:26 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:26 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:29:26 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:26 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:29:26 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:29:27 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:28 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:29:28 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:28 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:29:28 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:29:29 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:29 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:29:29 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:29 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:29:29 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:29:30 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:30 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:29:30 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:31 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:29:31 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:29:32 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:32 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:29:32 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:32 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:29:32 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:29:33 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:33 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:29:33 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:33 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:29:33 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:29:34 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:34 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:29:34 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:34 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:29:34 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:29:35 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:35 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:29:35 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:35 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:29:35 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:29:36 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:36 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:29:36 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:36 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:29:36 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:29:38 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:38 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:29:38 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:38 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:29:38 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:29:39 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:39 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:29:39 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:39 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:29:39 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:29:40 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:40 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:29:40 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:40 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:29:40 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:29:41 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:41 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:29:41 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:41 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:29:41 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:29:42 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:42 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:29:42 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:42 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:29:42 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:29:42 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:43 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:29:43 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:43 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:29:43 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:29:43 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:43 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:29:43 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:43 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:29:43 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:29:45 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:45 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:29:45 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:45 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:29:45 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:29:46 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:46 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:29:46 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:46 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 201 74
11/23/2018 05:29:46 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:46 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:46 PM http://127.0.0.1:8000 "POST /api/get-index/ HTTP/1.1" 200 12
11/23/2018 05:29:46 PM http://127.0.0.1:8000 "PATCH /api/lock_tokens/31b0c02704344e739d5639c56bf0147b HTTP/1.1" 200 74
11/23/2018 05:29:55 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:29:55 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 200 47
11/23/2018 05:29:55 PM {"error":"unexpected error, please try again."}
11/23/2018 05:29:55 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:29:55 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 200 47
11/23/2018 05:29:55 PM {"error":"unexpected error, please try again."}
11/23/2018 05:29:55 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:29:55 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 200 47
11/23/2018 05:29:55 PM {"error":"unexpected error, please try again."}
11/23/2018 05:29:55 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:29:55 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 200 47
11/23/2018 05:29:55 PM {"error":"unexpected error, please try again."}
11/23/2018 05:29:56 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:29:56 PM http://127.0.0.1:8000 "PATCH /api/lock_tokens/31b0c02704344e739d5639c56bf0147b HTTP/1.1" 200 74
11/23/2018 05:29:57 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:29:59 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 200 47
11/23/2018 05:29:59 PM {"error":"unexpected error, please try again."}
11/23/2018 05:30:25 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:30:25 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:30:25 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:30:25 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:30:25 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:30:26 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:30:27 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:30:27 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:30:27 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:30:27 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:30:28 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:30:28 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:30:28 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:30:28 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:30:28 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:30:39 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:30:39 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:30:39 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:30:39 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:30:39 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:30:40 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:30:40 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:30:40 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:30:40 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:30:40 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:30:41 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:30:42 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:30:42 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:30:42 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:30:42 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:30:43 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:30:43 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:30:43 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:30:43 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:30:43 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:30:44 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:30:44 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:30:44 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:30:44 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:30:44 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:30:46 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:30:46 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:30:46 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:30:46 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:30:46 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:30:47 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:30:47 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:30:47 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:30:47 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/23/2018 05:30:47 PM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/23/2018 05:30:59 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:31:00 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:31:00 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:31:00 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 201 74
11/23/2018 05:31:00 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:31:00 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:31:00 PM http://127.0.0.1:8000 "POST /api/get-index/ HTTP/1.1" 200 12
11/23/2018 05:31:00 PM http://127.0.0.1:8000 "PATCH /api/lock_tokens/8255b8a63e024c0fb0357c5d9a06b383 HTTP/1.1" 200 74
11/23/2018 05:31:02 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:31:02 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:31:02 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:31:02 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:31:02 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:31:02 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:31:02 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:31:02 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:31:04 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:31:07 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:31:07 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:31:07 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:31:07 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:31:07 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:31:07 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:31:07 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:31:07 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:31:07 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:31:07 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:31:07 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:31:07 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:31:07 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:31:07 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:31:07 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:31:07 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:31:07 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:31:07 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:31:07 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:31:07 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:31:08 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:31:08 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:31:08 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:31:08 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:31:08 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:31:08 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:31:08 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:31:08 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:31:08 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:31:08 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:31:08 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:31:08 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:31:08 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:31:08 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:31:08 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:31:08 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:31:08 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:31:08 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:31:08 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:31:08 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:31:08 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:31:08 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:31:08 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:31:08 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:31:08 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:31:08 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:31:08 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:31:08 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:31:08 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:31:08 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:31:09 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:31:09 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:31:09 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:31:09 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:31:09 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:31:09 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:31:09 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:31:09 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:31:09 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:31:09 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:31:09 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:31:09 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:31:09 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:31:09 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:31:09 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:31:09 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:31:09 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:31:09 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:31:09 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:31:09 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:31:09 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:31:09 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:31:09 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:31:09 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:31:09 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:31:09 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:31:09 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:31:09 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:31:09 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:31:09 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:31:10 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:31:10 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:31:10 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:31:10 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:31:10 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:31:10 PM http://127.0.0.1:8000 "PATCH /api/lock_tokens/8255b8a63e024c0fb0357c5d9a06b383 HTTP/1.1" 200 74
11/23/2018 05:31:10 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:31:10 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:31:10 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:31:10 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:31:10 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:31:10 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:31:10 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:31:10 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:31:10 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:31:10 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:31:10 PM http://127.0.0.1:8000 "DELETE /api/lock_tokens/8255b8a63e024c0fb0357c5d9a06b383 HTTP/1.1" 204 0
11/23/2018 05:32:45 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:32:45 PM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/23/2018 05:32:45 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:32:45 PM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 201 74
11/23/2018 05:32:45 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:32:45 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:32:45 PM http://127.0.0.1:8000 "POST /api/get-index/ HTTP/1.1" 200 5627
11/23/2018 05:32:45 PM http://127.0.0.1:8000 "PATCH /api/lock_tokens/0faf7886205d4a97a693d95f654d68eb HTTP/1.1" 200 74
11/23/2018 05:32:49 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:32:49 PM http://127.0.0.1:8000 "POST /api/delete/ HTTP/1.1" 200 24
11/23/2018 05:32:49 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:32:49 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:32:49 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:32:49 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:32:49 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:32:50 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:32:50 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:32:50 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:32:50 PM Resetting dropped connection: 127.0.0.1
11/23/2018 05:32:50 PM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/23/2018 05:32:50 PM Starting new HTTP connection (1): 127.0.0.1:8000
11/23/2018 05:32:50 PM http://127.0.0.1:8000 "DELETE /api/lock_tokens/0faf7886205d4a97a693d95f654d68eb HTTP/1.1" 204 0
11/24/2018 01:44:52 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:44:52 AM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/24/2018 01:44:52 AM 
Traceback (most recent call last):
  File "startup.py", line 55, in sync
    home_dir = check_home_dir()
  File "startup.py", line 183, in check_home_dir
    raise NoHomeDirException
NoHomeDirException
11/24/2018 01:45:19 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:45:19 AM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/24/2018 01:45:36 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:45:36 AM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 201 74
11/24/2018 01:45:36 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:45:36 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:45:36 AM http://127.0.0.1:8000 "POST /api/get-index/ HTTP/1.1" 200 12
11/24/2018 01:45:36 AM http://127.0.0.1:8000 "PATCH /api/lock_tokens/69ec96e3c4f44300a7fc77e0fb49e86c HTTP/1.1" 200 74
11/24/2018 01:45:39 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:45:39 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:45:39 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:45:39 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:45:39 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:45:39 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:45:39 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:45:39 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:45:39 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:45:39 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:45:39 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:45:39 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:45:39 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:45:39 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:45:41 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:45:45 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:45:45 AM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/24/2018 01:45:45 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:45:45 AM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/24/2018 01:45:45 AM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/24/2018 01:45:48 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:45:48 AM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/24/2018 01:45:48 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:45:48 AM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/24/2018 01:45:48 AM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/24/2018 01:45:50 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:45:50 AM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/24/2018 01:45:50 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:45:50 AM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/24/2018 01:45:50 AM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/24/2018 01:45:51 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:45:51 AM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/24/2018 01:45:51 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:45:51 AM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/24/2018 01:45:51 AM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/24/2018 01:45:52 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:45:52 AM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/24/2018 01:45:52 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:45:52 AM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/24/2018 01:45:52 AM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/24/2018 01:45:53 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:45:54 AM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/24/2018 01:45:54 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:45:54 AM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/24/2018 01:45:54 AM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/24/2018 01:45:54 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:45:56 AM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/24/2018 01:45:56 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:45:56 AM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/24/2018 01:45:56 AM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/24/2018 01:45:56 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:45:56 AM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/24/2018 01:45:56 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:45:56 AM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/24/2018 01:45:56 AM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/24/2018 01:45:58 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:45:58 AM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/24/2018 01:45:58 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:45:58 AM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/24/2018 01:45:58 AM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/24/2018 01:45:58 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:45:59 AM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/24/2018 01:45:59 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:45:59 AM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/24/2018 01:45:59 AM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/24/2018 01:45:59 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:45:59 AM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/24/2018 01:45:59 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:45:59 AM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/24/2018 01:45:59 AM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/24/2018 01:46:00 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:00 AM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/24/2018 01:46:00 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:00 AM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/24/2018 01:46:00 AM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/24/2018 01:46:01 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:01 AM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/24/2018 01:46:01 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:01 AM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/24/2018 01:46:01 AM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/24/2018 01:46:02 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:02 AM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/24/2018 01:46:02 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:02 AM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/24/2018 01:46:02 AM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/24/2018 01:46:02 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:02 AM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/24/2018 01:46:02 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:02 AM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/24/2018 01:46:02 AM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/24/2018 01:46:03 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:03 AM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/24/2018 01:46:03 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:03 AM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/24/2018 01:46:03 AM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/24/2018 01:46:04 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:04 AM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/24/2018 01:46:04 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:04 AM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/24/2018 01:46:04 AM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/24/2018 01:46:05 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:05 AM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/24/2018 01:46:05 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:05 AM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/24/2018 01:46:05 AM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/24/2018 01:46:06 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:06 AM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/24/2018 01:46:06 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:06 AM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/24/2018 01:46:06 AM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/24/2018 01:46:07 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:07 AM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/24/2018 01:46:07 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:07 AM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/24/2018 01:46:07 AM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/24/2018 01:46:07 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:07 AM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/24/2018 01:46:07 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:07 AM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/24/2018 01:46:07 AM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/24/2018 01:46:08 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:08 AM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/24/2018 01:46:08 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:08 AM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/24/2018 01:46:08 AM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/24/2018 01:46:09 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:09 AM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/24/2018 01:46:09 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:09 AM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/24/2018 01:46:09 AM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/24/2018 01:46:10 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:10 AM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/24/2018 01:46:10 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:10 AM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/24/2018 01:46:10 AM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/24/2018 01:46:11 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:11 AM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/24/2018 01:46:11 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:11 AM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/24/2018 01:46:11 AM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/24/2018 01:46:11 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:11 AM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/24/2018 01:46:11 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:11 AM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/24/2018 01:46:11 AM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/24/2018 01:46:12 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:12 AM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/24/2018 01:46:12 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:12 AM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/24/2018 01:46:12 AM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/24/2018 01:46:13 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:13 AM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/24/2018 01:46:13 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:13 AM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/24/2018 01:46:13 AM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/24/2018 01:46:14 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:14 AM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/24/2018 01:46:14 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:14 AM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/24/2018 01:46:14 AM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/24/2018 01:46:14 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:15 AM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/24/2018 01:46:15 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:15 AM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/24/2018 01:46:15 AM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/24/2018 01:46:15 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:15 AM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/24/2018 01:46:15 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:15 AM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/24/2018 01:46:15 AM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/24/2018 01:46:16 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:16 AM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/24/2018 01:46:16 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:16 AM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/24/2018 01:46:16 AM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/24/2018 01:46:17 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:17 AM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/24/2018 01:46:17 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:17 AM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/24/2018 01:46:17 AM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/24/2018 01:46:17 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:17 AM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/24/2018 01:46:17 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:17 AM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/24/2018 01:46:17 AM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/24/2018 01:46:19 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:19 AM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/24/2018 01:46:19 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:19 AM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/24/2018 01:46:19 AM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/24/2018 01:46:20 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:20 AM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/24/2018 01:46:20 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:20 AM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/24/2018 01:46:20 AM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/24/2018 01:46:21 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:21 AM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/24/2018 01:46:21 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:21 AM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/24/2018 01:46:21 AM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/24/2018 01:46:22 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:22 AM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/24/2018 01:46:22 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:22 AM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/24/2018 01:46:22 AM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/24/2018 01:46:23 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:23 AM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/24/2018 01:46:23 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:23 AM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/24/2018 01:46:23 AM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/24/2018 01:46:23 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:23 AM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/24/2018 01:46:23 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:23 AM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/24/2018 01:46:23 AM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/24/2018 01:46:24 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:25 AM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/24/2018 01:46:25 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:25 AM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/24/2018 01:46:25 AM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/24/2018 01:46:26 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:26 AM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/24/2018 01:46:26 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:26 AM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/24/2018 01:46:26 AM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/24/2018 01:46:27 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:27 AM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/24/2018 01:46:27 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:27 AM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/24/2018 01:46:27 AM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/24/2018 01:46:28 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:28 AM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/24/2018 01:46:28 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:28 AM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/24/2018 01:46:28 AM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/24/2018 01:46:29 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:29 AM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/24/2018 01:46:29 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:29 AM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/24/2018 01:46:29 AM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/24/2018 01:46:30 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:30 AM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/24/2018 01:46:30 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:30 AM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/24/2018 01:46:30 AM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/24/2018 01:46:31 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:31 AM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/24/2018 01:46:31 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:31 AM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/24/2018 01:46:31 AM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/24/2018 01:46:32 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:32 AM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/24/2018 01:46:32 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:32 AM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/24/2018 01:46:32 AM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/24/2018 01:46:33 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:33 AM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/24/2018 01:46:33 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:33 AM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/24/2018 01:46:33 AM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/24/2018 01:46:34 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:34 AM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/24/2018 01:46:34 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:34 AM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/24/2018 01:46:34 AM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/24/2018 01:46:35 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:35 AM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/24/2018 01:46:35 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:35 AM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 409 2
11/24/2018 01:46:35 AM 
Traceback (most recent call last):
  File "startup.py", line 57, in sync
    token = check_if_unlocked(server_url, AuthKey)
  File "startup.py", line 99, in check_if_unlocked
    raise requests.exceptions.HTTPError
requests.exceptions.HTTPError
11/24/2018 01:46:36 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:36 AM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/24/2018 01:46:36 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:36 AM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 201 74
11/24/2018 01:46:36 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:36 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:36 AM http://127.0.0.1:8000 "POST /api/get-index/ HTTP/1.1" 200 654
11/24/2018 01:46:36 AM http://127.0.0.1:8000 "PATCH /api/lock_tokens/609ec39b8372453290da57bfc386d63a HTTP/1.1" 200 74
11/24/2018 01:46:39 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:39 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:46:39 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:46:39 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:46:39 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:46:39 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:46:39 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:46:39 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:46:39 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:46:39 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:46:39 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:46:39 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:46:39 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:46:39 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:46:39 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:46:39 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:46:39 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:46:39 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:46:39 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:46:39 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:46:39 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:46:39 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:46:39 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:46:39 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:46:39 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:46:39 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:46:39 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:46:40 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:46:40 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:46:40 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:46:40 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:46:40 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:46:40 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:46:40 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:46:40 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:46:40 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:46:40 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:46:40 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:46:40 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:46:40 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:46:40 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:46:40 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:46:40 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:46:40 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:46:40 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:46:40 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:46:40 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:46:40 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:46:40 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:46:40 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:46:40 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:46:40 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:46:40 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:46:40 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:46:40 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:46:40 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:46:40 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:46:41 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:46:41 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:46:41 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:46:41 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:46:41 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:46:41 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:46:41 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:46:41 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:46:41 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:46:41 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:46:41 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:46:41 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:46:41 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:46:41 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:46:41 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:46:41 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:46:41 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:46:41 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:46:41 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:46:41 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:46:41 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:46:41 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:46:41 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:46:41 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:46:41 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:46:41 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:46:41 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:46:41 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:46:42 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:46:42 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:46:42 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:46:42 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:46:42 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:46:42 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:46:42 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:46:42 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:42 AM http://127.0.0.1:8000 "DELETE /api/lock_tokens/609ec39b8372453290da57bfc386d63a HTTP/1.1" 204 0
11/24/2018 01:46:52 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:52 AM http://127.0.0.1:8000 "POST /api/login/ HTTP/1.1" 200 50
11/24/2018 01:46:52 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:52 AM http://127.0.0.1:8000 "POST /api/lock_tokens/ HTTP/1.1" 201 74
11/24/2018 01:46:52 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:52 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:52 AM http://127.0.0.1:8000 "POST /api/get-index/ HTTP/1.1" 200 5853
11/24/2018 01:46:52 AM http://127.0.0.1:8000 "PATCH /api/lock_tokens/01f21f294ea244a88c9092d238313eec HTTP/1.1" 200 74
11/24/2018 01:46:56 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:56 AM http://127.0.0.1:8000 "POST /api/delete/ HTTP/1.1" 200 24
11/24/2018 01:46:56 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:56 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:46:56 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:46:56 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:46:56 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:46:56 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:46:56 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:46:56 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:46:56 AM Resetting dropped connection: 127.0.0.1
11/24/2018 01:46:56 AM http://127.0.0.1:8000 "POST /api/upload/ HTTP/1.1" 201 20
11/24/2018 01:46:56 AM Starting new HTTP connection (1): 127.0.0.1:8000
11/24/2018 01:46:56 AM http://127.0.0.1:8000 "DELETE /api/lock_tokens/01f21f294ea244a88c9092d238313eec HTTP/1.1" 204 0
